scripts/cent6_jswarm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+CINCH="$(readlink -f "$(dirname "$0")/../")"
+CINCH_VERSION=$(grep "${CINCH}/setup.py" -e 'version=' | sed -e "s/.*version='\(.*\)'.*/\1/")
+
+echo "*****************************************************"
+echo "Building cinch container for version ${CINCH_VERSION}"
+echo "*****************************************************"
+
+echo "Starting container"
+ansible -i /dev/null localhost -m docker_container -a "image=greghellings/centos_base:6 name=jswarm detach=true tty=true command=/bin/bash"
```

Comment:
Does Docker hub support team management of accounts?  This would be preferable to using a personal account to store the generated image.